Phx dev 9.5

honchung-tang opened the pull request:
## Description

Merge localization changes into phx-dev-9.5

yanji-hcl committed:
enable catalog server localization

yanji-hcl committed:
resource json structure

yanji-hcl committed:
Merge branch 'phx-dev' of https://github.com/HCL-TECH-SOFTWARE/volt-mx-tutorials into localization-enabelment

yanji-hcl committed:
another approach

yanji-hcl committed:
revoke unnecessary import

yanji-hcl committed:
add hindi

yanji-hcl committed:
Merge branch 'l10n' into localization-enabelment

testradav committed:
Merge pull request #13 from yanji-hcl/localization-enabelment

enable catalog server localization

yanji-hcl committed:
update translated tours jsons

yanji-hcl committed:
default redirect to en

yanji-hcl committed:
remove dup item

yanji-hcl committed:
fix loading in webview

mikeangelsilva committed:
Merge pull request #28 from yanji-hcl/translated-tours

update translated tours jsons

mikeangelsilva committed:
set to phx-dev

mikeangelsilva committed:
Merge pull request #30 from mikeangelsilva/update

set to phx-dev

honlamso committed:
Merge commit '94fc0fd70825ec49a16279a58f4bbce0a222f01e' into l10n

# Conflicts:
#	.github/workflows/intergrate.yml
#	next.config.js
#	pages/hikes.js
#	pages/tour.js
#	public/contents/1-volt-mx-changes/tours.json
#	public/contents/1-volt-mx-changes/zips/voltmxirisintro.zip
#	public/contents/2-build-your-first-mobile-app/tours.json
#	public/contents/2-build-your-first-mobile-app/zips/addcreatemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/addloginservicemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/connectappuitodatamob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createdetailsnative_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createservices_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createuifromcollectionsmob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/editpayeemobile_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/launchyourfirstmobileappv92.zip
#	public/contents/2-build-your-first-mobile-app/zips/pubishappenterprisestorev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/shareandpreviewappmobilev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/usingtestrecorder_v9.zip
#	public/contents/3-build-your-first-web-app/tours.json
#	public/contents/3-build-your-first-web-app/zips/addauthenticationtoyourapp_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addcreatepayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/connecttheappuitodata-webv9.zip
#	public/contents/3-build-your-first-web-app/zips/createdetailsscreen_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createservicesweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createuifromcollections-web.zip
#	public/contents/3-build-your-first-web-app/zips/editpayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/launchyourrwapp92.zip
#	public/contents/3-build-your-first-web-app/zips/responsivewebbreakpointsv9.zip
#	public/contents/4-develop-the-front-end/tours.json
#	public/contents/4-develop-the-front-end/zips/assigndatatothesegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/createemailidcomponentv9.zip
#	public/contents/4-develop-the-front-end/zips/flexpropertiesv9.zip
#	public/contents/4-develop-the-front-end/zips/internationalizationwithlayout92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontoactioneditorv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontointernationalization92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontokonyjavascriptv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmenttemplatesv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/introflexlayoutv9.zip
#	public/contents/4-develop-the-front-end/zips/konyquantumorientationv9.zip
#	public/contents/4-develop-the-front-end/zips/understandingapplicationchannelsv9.zip
#	public/contents/5-manage-back-end-services/tours.json
#	public/contents/5-manage-back-end-services/zips/addasalesforceservicev9.zip
#	public/contents/5-manage-back-end-services/zips/understandidentityservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/userrepositoryauthenticationv9.zip
#	public/contents/5-manage-back-end-services/zips/usesampleservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/usestorageservicev9.zip
#	public/contents/6-advanced-concepts/tours.json
#	public/contents/6-advanced-concepts/zips/introductiontonfiv9.zip
#	public/contents/6-advanced-concepts/zips/nfibarcodev9.zip
#	public/contents/7-dbx/tours.json
#	public/contents/7-dbx/zips/dynamicform.zip
#	src/components/HikeHeader/index.js
#	src/components/TourCard/index.js
#	src/utils/populate.js

yanji-hcl committed:
change to i10n for deploy

yanji-hcl committed:
rollback

yanji-hcl committed:
add log before reload

yanji-hcl committed:
disable reload

yanji-hcl committed:
fix continues reloading

yanji-hcl committed:
revoke typo

yanji-hcl committed:
Merge phx-dev

yanji-hcl committed:
add tours json back

mikeangelsilva committed:
Merge pull request #42 from yanji-hcl/l10n

Merge latest phx-dev to l10n

yanji-hcl committed:
Merge branch 'l10n' of https://github.com/HCL-TECH-SOFTWARE/volt-mx-tutorials into fix-empty-router-query

yanji-hcl committed:
get hike content when switch lang

testradav committed:
Merge pull request #41 from yanji-hcl/fix-empty-router-query

Fix continues loading in gh pages env

honlamso committed:
composer edit category in different locales

honlamso committed:
translations

honlamso committed:
update push mechanism

honlamso committed:
Merge commit 'a8312785f2312214c70465fbd13ee55276ee7cd7' into HPHX5988

# Conflicts:
#	public/contents/2-build-your-first-mobile-app/zips/addcreatemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/addloginservicemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/connectappuitodatamob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createdetailsnative_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createservices_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createuifromcollectionsmob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/editpayeemobile_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/launchyourfirstmobileappv92.zip
#	public/contents/2-build-your-first-mobile-app/zips/pubishappenterprisestorev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/shareandpreviewappmobilev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/usingtestrecorder_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addauthenticationtoyourapp_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addcreatepayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/connecttheappuitodata-webv9.zip
#	public/contents/3-build-your-first-web-app/zips/createdetailsscreen_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createservicesweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createuifromcollections-web.zip
#	public/contents/3-build-your-first-web-app/zips/editpayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/launchyourrwapp92.zip
#	public/contents/3-build-your-first-web-app/zips/responsivewebbreakpointsv9.zip
#	public/contents/4-develop-the-front-end/zips/createemailidcomponentv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmenttemplatesv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/understandingapplicationchannelsv9.zip
#	public/contents/5-manage-back-end-services/zips/understandidentityservicesv9.zip
#	public/contents/6-advanced-concepts/zips/introductiontonfiv9.zip
#	public/contents/6-advanced-concepts/zips/nfibarcodev9.zip

honlamso committed:
Merge commit 'e1007476ec2d37b83c1d393fd4ee7faabf9f0344' into HPHX5988

# Conflicts:
#	public/contents/2-build-your-first-mobile-app/zips/addloginservicemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/connectappuitodatamob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createuifromcollectionsmob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/launchyourfirstmobileappv92.zip
#	public/contents/3-build-your-first-web-app/zips/addauthenticationtoyourapp_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createservicesweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createuifromcollections-web.zip
#	public/contents/3-build-your-first-web-app/zips/launchyourrwapp92.zip
#	public/contents/7-dbx/tours.json
#	public/contents/7-dbx/zips/dynamicform.zip

honlamso committed:
Merge branch 'l10n' into HPHX5988b

# Conflicts:
#	package-lock.json
#	pages/hikes.js
#	pages/tour.js
#	public/contents/1-volt-mx-changes/tours.json
#	public/contents/2-build-your-first-mobile-app/tours.json
#	public/contents/2-build-your-first-mobile-app/zips/addcreatemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/addloginservicemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/connectappuitodatamob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createdetailsnative_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createservices_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createuifromcollectionsmob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/editpayeemobile_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/launchyourfirstmobileappv92.zip
#	public/contents/2-build-your-first-mobile-app/zips/pubishappenterprisestorev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/shareandpreviewappmobilev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/usingtestrecorder_v9.zip
#	public/contents/3-build-your-first-web-app/tours.json
#	public/contents/3-build-your-first-web-app/zips/addauthenticationtoyourapp_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addcreatepayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/connecttheappuitodata-webv9.zip
#	public/contents/3-build-your-first-web-app/zips/createdetailsscreen_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createservicesweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createuifromcollections-web.zip
#	public/contents/3-build-your-first-web-app/zips/editpayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/launchyourrwapp92.zip
#	public/contents/3-build-your-first-web-app/zips/responsivewebbreakpointsv9.zip
#	public/contents/4-develop-the-front-end/zips/assigndatatothesegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/createemailidcomponentv9.zip
#	public/contents/4-develop-the-front-end/zips/flexpropertiesv9.zip
#	public/contents/4-develop-the-front-end/zips/internationalizationwithlayout92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontoactioneditorv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontointernationalization92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmenttemplatesv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/introflexlayoutv9.zip
#	public/contents/4-develop-the-front-end/zips/understandingapplicationchannelsv9.zip
#	public/contents/5-manage-back-end-services/zips/addasalesforceservicev9.zip
#	public/contents/5-manage-back-end-services/zips/understandidentityservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/userrepositoryauthenticationv9.zip
#	public/contents/5-manage-back-end-services/zips/usesampleservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/usestorageservicev9.zip
#	public/contents/6-advanced-concepts/zips/introductiontonfiv9.zip
#	public/contents/6-advanced-concepts/zips/nfibarcodev9.zip
#	public/contents/7-dbx/tours.json
#	public/contents/7-dbx/zips/dynamicform.zip
#	src/components/HikeHeader/index.js
#	src/components/TourCard/index.js
#	src/utils/populate.js

honlamso committed:
Merge branch 'HPHX5988' into HPHX5988b

# Conflicts:
#	pages/hikelisting.js
#	pages/hikes.js
#	pages/tour.js
#	public/contents/1-volt-mx-changes/tours.json
#	public/contents/1-volt-mx-changes/tours_en-US.json
#	public/contents/1-volt-mx-changes/tours_fr-FR.json
#	public/contents/1-volt-mx-changes/tours_hi-IN.json
#	public/contents/1-volt-mx-changes/tours_zh-CN.json
#	public/contents/1-volt-mx-changes/tours_zh-TW.json
#	public/contents/1-volt-mx-changes/zips/voltmxirisintro.zip
#	public/contents/2-build-your-first-mobile-app/zips/addcreatemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/addloginservicemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/connectappuitodatamob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createdetailsnative_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createservices_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createuifromcollectionsmob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/editpayeemobile_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/launchyourfirstmobileappv92.zip
#	public/contents/2-build-your-first-mobile-app/zips/pubishappenterprisestorev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/shareandpreviewappmobilev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/usingtestrecorder_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addauthenticationtoyourapp_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addcreatepayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/connecttheappuitodata-webv9.zip
#	public/contents/3-build-your-first-web-app/zips/createdetailsscreen_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createservicesweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createuifromcollections-web.zip
#	public/contents/3-build-your-first-web-app/zips/editpayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/launchyourrwapp92.zip
#	public/contents/3-build-your-first-web-app/zips/responsivewebbreakpointsv9.zip
#	public/contents/4-develop-the-front-end/zips/assigndatatothesegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/createemailidcomponentv9.zip
#	public/contents/4-develop-the-front-end/zips/flexpropertiesv9.zip
#	public/contents/4-develop-the-front-end/zips/internationalizationwithlayout92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontoactioneditorv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontointernationalization92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontokonyjavascriptv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmenttemplatesv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/introflexlayoutv9.zip
#	public/contents/4-develop-the-front-end/zips/konyquantumorientationv9.zip
#	public/contents/4-develop-the-front-end/zips/understandingapplicationchannelsv9.zip
#	public/contents/5-manage-back-end-services/zips/addasalesforceservicev9.zip
#	public/contents/5-manage-back-end-services/zips/understandidentityservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/userrepositoryauthenticationv9.zip
#	public/contents/5-manage-back-end-services/zips/usesampleservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/usestorageservicev9.zip
#	public/contents/6-advanced-concepts/zips/introductiontonfiv9.zip
#	public/contents/6-advanced-concepts/zips/nfibarcodev9.zip
#	public/contents/7-dbx/tours.json
#	public/contents/7-dbx/tours_en-US.json
#	public/contents/7-dbx/tours_fr-FR.json
#	public/contents/7-dbx/tours_hi-IN.json
#	public/contents/7-dbx/tours_zh-CN.json
#	public/contents/7-dbx/tours_zh-TW.json
#	public/contents/7-dbx/zips/dynamicform.zip
#	src/components/TourCard/index.js
#	src/utils/populate.js

honlamso committed:
fix translations

yanji-hcl committed:
Merge branch 'phx-dev' of https://github.com/HCL-TECH-SOFTWARE/volt-mx-tutorials into sync-phx-dev

mikeangelsilva committed:
Merge pull request #48 from yanji-hcl/sync-phx-dev

Sync phx dev

jkjhcl committed:
translated the composer page and update the hikes content translated

mikeangelsilva committed:
Merge pull request #49 from jkjhcl/l10n

translated the composer page and update the hikes content translated

honlamso committed:
Merge branch 'HPHX5988b' into HPHX5988

# Conflicts:
#	package-lock.json
#	pages/composer.js
#	pages/hikes.js
#	pages/tour.js
#	public/contents/2-build-your-first-mobile-app/tours.json
#	public/contents/2-build-your-first-mobile-app/tours_en-US.json
#	public/contents/2-build-your-first-mobile-app/tours_fr-FR.json
#	public/contents/2-build-your-first-mobile-app/tours_hi-IN.json
#	public/contents/2-build-your-first-mobile-app/tours_zh-CN.json
#	public/contents/2-build-your-first-mobile-app/tours_zh-TW.json
#	public/contents/2-build-your-first-mobile-app/zips/addcreatemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/addloginservicemob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/connectappuitodatamob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createdetailsnative_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createservices_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/createuifromcollectionsmob_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/editpayeemobile_v9.zip
#	public/contents/2-build-your-first-mobile-app/zips/launchyourfirstmobileappv92.zip
#	public/contents/2-build-your-first-mobile-app/zips/pubishappenterprisestorev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/shareandpreviewappmobilev9.zip
#	public/contents/2-build-your-first-mobile-app/zips/usingtestrecorder_v9.zip
#	public/contents/3-build-your-first-web-app/tours.json
#	public/contents/3-build-your-first-web-app/tours_en-US.json
#	public/contents/3-build-your-first-web-app/tours_fr-FR.json
#	public/contents/3-build-your-first-web-app/tours_hi-IN.json
#	public/contents/3-build-your-first-web-app/tours_zh-CN.json
#	public/contents/3-build-your-first-web-app/tours_zh-TW.json
#	public/contents/3-build-your-first-web-app/zips/addauthenticationtoyourapp_v9.zip
#	public/contents/3-build-your-first-web-app/zips/addcreatepayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/connecttheappuitodata-webv9.zip
#	public/contents/3-build-your-first-web-app/zips/createdetailsscreen_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createservicesweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/createuifromcollections-web.zip
#	public/contents/3-build-your-first-web-app/zips/editpayeeweb_v9.zip
#	public/contents/3-build-your-first-web-app/zips/launchyourrwapp92.zip
#	public/contents/3-build-your-first-web-app/zips/responsivewebbreakpointsv9.zip
#	public/contents/4-develop-the-front-end/tours_en-US.json
#	public/contents/4-develop-the-front-end/tours_fr-FR.json
#	public/contents/4-develop-the-front-end/tours_hi-IN.json
#	public/contents/4-develop-the-front-end/tours_zh-CN.json
#	public/contents/4-develop-the-front-end/tours_zh-TW.json
#	public/contents/4-develop-the-front-end/zips/assigndatatothesegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/createemailidcomponentv9.zip
#	public/contents/4-develop-the-front-end/zips/flexpropertiesv9.zip
#	public/contents/4-develop-the-front-end/zips/internationalizationwithlayout92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontoactioneditorv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontointernationalization92.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmenttemplatesv9.zip
#	public/contents/4-develop-the-front-end/zips/introductiontosegmentv9.zip
#	public/contents/4-develop-the-front-end/zips/introflexlayoutv9.zip
#	public/contents/4-develop-the-front-end/zips/understandingapplicationchannelsv9.zip
#	public/contents/5-manage-back-end-services/tours_en-US.json
#	public/contents/5-manage-back-end-services/tours_fr-FR.json
#	public/contents/5-manage-back-end-services/tours_hi-IN.json
#	public/contents/5-manage-back-end-services/tours_zh-CN.json
#	public/contents/5-manage-back-end-services/tours_zh-TW.json
#	public/contents/5-manage-back-end-services/zips/addasalesforceservicev9.zip
#	public/contents/5-manage-back-end-services/zips/understandidentityservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/userrepositoryauthenticationv9.zip
#	public/contents/5-manage-back-end-services/zips/usesampleservicesv9.zip
#	public/contents/5-manage-back-end-services/zips/usestorageservicev9.zip
#	public/contents/6-advanced-concepts/tours_en-US.json
#	public/contents/6-advanced-concepts/tours_fr-FR.json
#	public/contents/6-advanced-concepts/tours_hi-IN.json
#	public/contents/6-advanced-concepts/tours_zh-CN.json
#	public/contents/6-advanced-concepts/tours_zh-TW.json
#	public/contents/6-advanced-concepts/zips/introductiontonfiv9.zip
#	public/contents/6-advanced-concepts/zips/nfibarcodev9.zip
#	public/locales/en-US/translation.json
#	public/locales/hi-IN/translation.json
#	public/locales/zh-CN/translation.json
#	public/locales/zh-TW/translation.json
#	src/utils/populate.js

honlamso committed:
fix merge issue

mikeangelsilva committed:
Merge pull request #50 from honlamso/HPHX5988

HPHX5988 - Editing tours.json in different language in composer page

yanji-hcl committed:
switch lang effective immediately

testradav committed:
Merge pull request #51 from yanji-hcl/switch-lang-not-work-first-time

HPHX-6481: Switch lang does not work the first time

jkjhcl committed:
Changes some translation

jkjhcl committed:
Changes some translation

mikeangelsilva committed:
Merge pull request #52 from jkjhcl/l10n

Update the translation words for tours content

honlamso committed:
merge with master

honlamso committed:
fix translations

honlamso committed:
fix search; add back missing strings

honlamso committed:
Merge branch 'l10n'

# Conflicts:
#	src/components/TourCard/index.js

honlamso committed:
Merge commit '61b9b964c7844a9a99c278456a6e71f17f3cb458' into l10n

# Conflicts:
#	src/components/TourCard/index.js

honlamso committed:
chinese translations

honlamso committed:
Merge branch 'l10n'

# Conflicts:
#	src/components/TourCard/index.js

honlamso committed:
configs update for gh-pages

honlamso committed:
fix url

honlamso committed:
fix search

honlamso committed:
Merge branch 'l10n'

honlamso committed:
debug logs

honlamso committed:
fix keyword undefined

honlamso committed:
attempt to fix search

honlamso committed:
revert debug changes

honlamso committed:
fix changePreview logic, remove commented code

honlamso committed:
dependencies update

honchung-tang committed:
Update README.md

honchung-tang committed:
Update README.md

honchung-tang committed:
Update hikes content from master

honchung-tang committed:
Update README.md

Trigger build and deploy

honchung-tang committed:
Update intergrate.yml

honchung-tang committed:
Update environment config for testing

honchung-tang committed:
Merge branch 'l10n' of github.com:honlamso/volt-mx-tutorials into l10n

honchung-tang committed:
HPHX-22709 - Fix the symbol

honchung-tang committed:
Merge branch 'l10n' into phx-dev-9.5

honchung-tang committed:
Update content from HCL-TECH-SOFTWARE:phx-dev-9.5

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/HCL-TECH-SOFTWARE/volt-mx-tutorials?pullRequest=162) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/HCL-TECH-SOFTWARE/volt-mx-tutorials?pullRequest=162) before we can accept your contribution.<br/>**6** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: yanji-hcl<br/>:white_check_mark: testradav<br/>:white_check_mark: jkjhcl<br/>:white_check_mark: honlamso<br/>:white_check_mark: mandeepalluru<br/>:white_check_mark: mikeangelsilva<br/>:x: honchung-tang<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/HCL-TECH-SOFTWARE/volt-mx-tutorials?pullRequest=162) it.</sub>

honchung-tang committed:
Update hikeBuilder.zip with localization changes (HPHX-15372)

mandeepalluru committed:
HPHX-20103: Hide language selection dropdown on Hike Viewer

    Introduced setting named "enableLocales".

    enableLocales = true => Language selection dropdown is shown
                  = false => Language selection dropdown is NOT shown

honchung-tang committed:
Merge pull request #1 from mandeepalluru/HPHX-20103

HPHX-20103: Hide language selection dropdown on Hike Viewer

honchung-tang committed:
HPHX-24124
HPHX-24122
HPHX-24123
HPHX-24115
HPHX-24110
HPHX-24119
HPHX-24125

honchung-tang committed:
HPHX-24100

honchung-tang committed:
HPHX-24125

honchung-tang committed:
HPHX-24213

honchung-tang committed:
HPHX-24125
HPHX-24233

honchung-tang committed:
HPHX-23864
